pi_server/setup.py

```diff
@@ -4,11 +4,11 @@
     name="pi_server",
     version="0.1.0",
     packages=find_packages(exclude=["tests"]),
-    install_requires=["flask==2.0.3"],
-    extras_require={"dev": ["pytest", "mypy", "black", "isort"]},
+    install_requires=["flask==2.0.3"],#, "RPi.GPIO==0.7.1"],
```

Comment:
Uncomment that since I think it's useful

Reply:
Solved